README.md

```diff
@@ -55,7 +55,7 @@ $temporaryDirectory->path('dumps/datadump.dat'); // return  /full/path/to/tempor
 
 ### Deleting a temporary folder
 
-Once you're done processing your temporary data you can delete the entire temporary directory using the `delete` method. All files inside of it will be deleted. 
```

Comment:
Ohhh my bad makes sense for markdown, since trailing spaces can have meaning. We can leave them trimmed here though.